### PR TITLE
Merge staging v1.8.0 into main branch

### PR DESCRIPTION
**Pull request by fernandofloresg**
##### SUMMARY

Part of the release process, merge v1.8.0 into main.

##### ISSUE TYPE

- Release Pull Request

### CODE REVIEW

**Review by richp405 — APPROVED** [review]
looks good to me

**Review by rexemin — APPROVED** [review]
Looks good and complete!

File: plugins/action/zos_job_submit.py [hunk 1]
```diff
@@ -60,6 +60,11 @@ def run(self, tmp=None, task_vars=None):
             dest_path = self._execute_module(
                 module_name="tempfile", module_args={}, task_vars=task_vars,
             ).get("path")
+            # Calling execute_module from this step with tempfile leaves behind a tmpdir.
```
Comment (ddimatos) [inline plugins/action/zos_job_submit.py:63]: Nice job on this one, for catching it and fixing and porting it back to dev :)

**Review by AndreMarcel99 — APPROVED** [review]

**Review by ddimatos — APPROVED** [review]
This looks great, nice work @fernandofloresg

**Pull request merged**